src/ansys/units/systems.py

```diff
@@ -193,14 +184,25 @@ def CURRENT(self):
     def CURRENT(self, new_mass):
         self._set_type(unit_type=BaseDimensions.CURRENT, unit=new_mass)
 
-    @property
-    def SOLID_ANGLE(self):
-        """Solid Angle unit of the unit system."""
-        return self._SOLID_ANGLE
+    if os.getenv("PYANSYS_UNITS_ANGLE_AS_DIMENSION"):
```

Comment:
Need to discuss.

Reply:
The idea was that we would allow the constraint regarding arithmetic operations between angles and dimensionless variables to be lifted. This says nothing about lifting support for angle units, which we can still do independently of which arithmetic operations are allowed.

Reply:
@seanpearsonuk 

Ok. I just understood your concern.

```
>>> si = UnitSystem()
>>> deg = Quantity(11.2, "degree")
>>> deg_si = deg.convert(si) 
>>> deg_si
Quantity (0.1954768762233649, "")     # should be Quantity (0.1954768762233649, "radian") 
>>> deg_si.value
0.1954768762233649
>>> deg_si.units
_name:                                    # should be _name:  radian
_dimensions:
_si_units:
_si_scaling_factor: 1.0
_si_offset: 0.0
```

Reply:
We are not handling at this level.